src/wp-admin/edit-form-blocks.php

```diff
@@ -305,7 +305,7 @@
  * @param string  $text Placeholder text. Default 'Start writing or type / to choose a block'.
```

Comment:
Can you please update the doc block as per the new message?

`* @param string  $text Placeholder text. Default 'Start writing or type / to choose a block'.`
Replace to
`* @param string  $text Placeholder text. Default 'Type / to choose a block'.`